source/Tutorials/Advanced/Simulation-Webots/Installation-MacOS.rst

```diff
@@ -212,19 +212,20 @@ If Webots is closed or the ROS 2 process is interrupted, the local server will a
 Pre-configured Images
 -----------------------
 
-The following links provide you with pre-configured UTM images for each version of Webots. 
+If you don't want to setup the VM from scratch, the following links provide you with pre-configured UTM images for each version of Webots. 
 The ``webots_ros2`` version is installed from the official repository (not from sources) and is typically the first one that is compatible with the corresponding Webots version. 
 You are welcome to download an image and upgrade the package, or install it from sources if necessary.
 
-* R2023a
+* `Version 2023.0.2 for Webots R2023a <https://cyberbotics.com/files/ros2/webots_ros2_2023_0_2.utm.zip>`_
```

Comment:
Since it is a very big file, it would be nice to display its size so that the user is warned that she is about to download a big chunk.